MapboxNavigation/Constants.swift

```diff
@@ -30,8 +30,3 @@ public let MBCurrentLegAttribute = "isCurrentLeg"
  Attribute name for the route line that is used for identifying different `CongestionLevel` along the route.
  */
 public let MBCongestionAttribute = "congestion"
-
-/**
- The minimum distance of a tunnel intersection on a given route step.
- */
-public var MBMinimumTunnelLengthForUsingNightStyle: CLLocationDistance = 100
```

Comment:
I think the intent was to address the feedback in https://github.com/mapbox/mapbox-navigation-ios/pull/1127#discussion_r172624263 by moving this constant, but I don’t see where it got moved to.

Reply:
@1ec5 we're not using this constant at the moment. The tunnel distance check will be done on my upcoming PR for simulation of the tunnel navigation.